scripts/mixins/families/antlion_ambush.lua

```diff
@@ -2,6 +2,8 @@
 
 require('scripts/globals/mixins')
 
+local pitAmbush = 278
```

Comment:
Define this inside the listener its actually used in

Reply:
it's used in two

Reply:
Edit: done